sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero_mem.cpp

```diff
@@ -1680,6 +1680,9 @@ UR_APIEXPORT ur_result_t UR_APICALL urMemImageCreateWithNativeHandle(
       return Res;
     }
     Image->ZeImageDesc = ZeImageDesc;
+#else
+    std::ignore = ImageFormat;
+    std::ignore = ImageDesc;
```

Comment:
thanks @aelovikov-intel. right, seems we have two patches doing the same.

since we need this to unblock internal testing, and https://github.com/intel/llvm/pull/9655/files has other changes, I will cherry-pick this to internal and then we can think on removing my changes later.

@aelovikov-intel, @smaslov-intel: would that work?

Reply:
I'm fine either way.

Reply:
I think we can keep both changes